.devops/nix/package.nix

```diff
@@ -200,7 +200,7 @@ effectiveStdenv.mkDerivation (
       ++ optionals useMpi [ mpi ]
       ++ optionals useOpenCL [ clblast ]
       ++ optionals useRocm rocmBuildInputs
-      ++ optionals useBlas [ blas ]
+      ++ optionals (useBlas && blas.meta.available) [ blas ]
```

Comment:
Also I think we should instead `&&` this in https://github.com/ggerganov/llama.cpp/blob/ae316601c60a880d255d877f167eae7a58cc2e05/.devops/nix/package.nix#L21-L27.

Rationale: user-supplied `useBlas` should have the priority, but we want to choose a different default for `useBlas` in the cross-compilation setting